### ISSUE TITLE
Rework conflict resolution framework for Mozilla's communities

### ISSUE THREAD

**Issue by deimidis** [mozilla/participation-org#270]
## Goal:

Mozilla communities needs a way to handle conflict inside themselves and a proper way to escalate it in case they couldn't resolve it
## Success:

Create a proposal that could be discussed with stakeholders outside Participation
## Related Issues & Links:
#171
## Roles:

Owner: @deimidis 
Involved: @rosanardila, @nukeador, Larissa Shapiro
## Required:
- [ ] Evaluate Mozilla's past system ([Conductors program](https://wiki.mozilla.org/Conductors)) (@deimidis, [medium], Due: Dec 30th) 
- [ ] Research for community current solutions (@deimidis, [small], Due: Jan 8th) 
- [ ] Draft about process and framework for conflict resolution (@deimidis, [large], Due: Jan 8th)

**Comment by LucyeoH-zz** [comment]
Notes: Build a session for the leadership summit on conflict resolution.

**Comment by deimidis** [comment]
Leadership Summit session was created by Sara Haghdoosti

**Comment by ioana-chiorean** [comment]
The session in the Leadership Summit was more like a Share & tell - each one of the participants shared w bit from their own history,. We formed groups and identify several conflicts. We tried also to see if they were fixed or not - one thing was in all groups - we tend to avoid fixing conflicts and we "kill" volunteers by letting them burn in conflict. Sorry for the strong words but  we had a big AHA moment when we realized that the biggest conflicts were never solved.  A big **missing piece** was to share how to deal with these situation. (Here are my notes: https://www.flickr.com/photos/ioana_cis/24778676072/in/album-72157664287771411/)

Unfortunately the structure here was not met: https://wiki.mozilla.org/Participation/Global_Gatherings_2015/Singapore/Content/conflict_resolution ( also has miss-pointing links)

Would love to see what Sara prepared and also her notes. 
Would be nice to keep this as a subject to be developer further more.

cc @soapdog @dricupello @rosanardila
